DuckDuckGo/TabSwitcherViewController+MultiSelect.swift

```diff
@@ -345,8 +251,15 @@ extension TabSwitcherViewController {
         }.count
     }
 
+    var allPagesCount: Int {
+        return tabsModel.tabs.compactMap {
+            $0.link
+        }.count
```

Comment:
You could also do a tidy one-liner
```suggestion
        tabsModel.tabs.compactMap(\.link).count
```
and skip `return`.